README.md

```diff
@@ -1,106 +1,68 @@
-# iob-cache
```

Comment:
Lines 1-5 can't be removed

Reply:
If there is a publication it is the first thing to mention after the title

Reply:
OK